### DISCUSSION TITLE
Clicking the (1) in repositories filter shouldn't clear the filter

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44030] [Mobile] [answered]
### Select Topic Area

Bug

### Body

GitHub Mobile Beta

Visit repositories and filter by fork:
![image](https://user-images.githubusercontent.com/2119212/211939042-7106db37-e589-4f2d-ade8-a8dd706787ef.png)

I asked someone what they expected clicking the `⛛(1)` would do... Their reasonable guess:

> hitting the (1) will let you select filters and sort order

### Actual results

It clears the search

### other places behave closer to expected

Pull requests:
![Screenshot_20230111-190544.png](https://user-images.githubusercontent.com/2119212/211944626-59df17f4-f9df-4490-bc88-9c23935d73a1.png)

**Comment by HenSquared** [comment, marked as answer]
Thanks for the feedback. I've opened an issue with the team to get this to align with expected behavior.
